added buttons to detailed results page

GeoZez opened the pull request.

benlikescode commented:
Hey, thanks for contributing! Not sure how I feel about the buttons being there. Even if you align them right, it gets a little crowded and you'd have to consider removing/stacking them on mobile. 

IMO maybe just a link where the map name already is with like a underline on hover would be better?

GeoZez commented:
I removed the buttons and added links to the map avatar and the map name. If anything should be changed to be more consistent with how you would do things, please do.

benlikescode commented on components/Results/LeaderboardCard/LeaderboardCard.Styled.tsx:
```diff
@@ -67,9 +67,13 @@ const StyledLeaderboardCard = styled.div`
           gap: 5px;
 
-          .label1 {
+          .mapNameLabel, label1 {
```
`label1` also needs a period in front like `.label1`
